frontend/src/components/commons/BackToTop.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import LOCALIZE from "../../text_resources";
+import scrollToTop from "../../helpers/scrollToTop";
+
+const styles = {
+  displayedButton: {
+    display: "block",
+    position: "fixed",
+    bottom: "5%",
```

Comment:
can we make these pixels instead of percents? 5% on a large screen vs a small one is actually very different amount of white space. 20px each should be good.

Reply:
I'll take care of this in my next PR once you merge.